src/sql/workbench/contrib/query/browser/queryResultsView.ts

```diff
@@ -209,7 +209,10 @@ export class QueryResultsView extends Disposable {
 	}
 
 	private setQueryRunner(runner: QueryRunner) {
```

Comment:
It would be great if you took the time to do cleanup stuff like add return types when going through older code